Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -251,7 +251,7 @@ - (void)startLoading {
 
         NSError* error = nil;
         try {
-            auto httpRequestMessage = _requestMessageForNS(_request);
+            auto httpRequestMessage = _requestMessageForNS(self.request);
```

Comment:
these getter calls seem unnecessary; NSURLProtocol for HTTP is a leaf class and cannot be subclassed.